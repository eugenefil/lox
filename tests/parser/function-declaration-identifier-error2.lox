fn f(x, 5) {}
