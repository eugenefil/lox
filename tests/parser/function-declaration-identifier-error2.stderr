error: expected identifier
 --> $DIR/function-declaration-identifier-error2.lox:1:9
  |
1 | fn f(x, 5) {}
  |         ^
